### DISCUSSION TITLE
Half pie chart. How to?

### DISCUSSION THREAD

**Discussion by calinciubotariublankfactor** [Abhinandan-Kushwaha/react-native-gifted-charts#616] [Q&A]
I cannot find in the docs how to plot a half pie chart (only 180* angle).
How can I achieve something like this?
![React Native Half Pie Chart](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/156336682/ea63a422-aae1-415e-b0e3-9a4537ce23ea)

Thanks!

**Comment by Abhinandan-Kushwaha** [comment]
Use the prop `semicircle` in `<PieChart>` components. Then use `donut` prop.
